packages/twenty-front/src/modules/ui/navigation/menu-item/components/MenuItemMultiSelect.tsx

```diff
@@ -36,7 +39,7 @@ export const MenuItemMultiSelect = ({
     <StyledMenuItemBase className={className} onClick={handleOnClick}>
       <StyledLeftContentWithCheckboxContainer>
         <Checkbox checked={selected} />
-        <MenuItemLeftContent LeftIcon={LeftIcon} text={text} />
+        <Tag color={color} text={text} Icon={LeftIcon} />
```

Comment:
We should keep the previous behavior if no color is passed.

Reply:
Hey @lucasbordeau, I've made the required changes. Please check if it's alright now.